mandatory/1-fix-functions.js

```diff
@@ -10,18 +10,18 @@
 
 */
 
-function mood() {
-  let isHappy = true;
+function mood(isHappy) {
+  // let isHappy = true;
 
-  if (isHappy) {
+  if (isHappy === true) {
```

Comment:
Hi Seyed, I think you can spare the (isHappy === true) bit since you have isHappy as your parameter in the mood function. You can just write if(isHappy), it will return true anyway. Other your code is nice and well-indented.

Reply:
Thank you .